recipe/meta.yaml

```diff
@@ -20,38 +17,48 @@ source:
     - gvconfig_libdir.patch  # [unix]
 
 build:
-  number: 7
+  number: 8
   detect_binary_files_with_prefix: True  # [unix]
 
 requirements:
   build:
-    - cairo 1.14.*             # [linux]
-    - expat 2.2.*                   # [linux]
-    - libpng >=1.6.32,<1.6.35  # [unix]
-    - libtiff >=4.0.8,<4.0.10   # [unix]
-    - libtool                  # [linux]
-    - freetype 2.8.1             # [unix]
-    - jpeg 9*                  # [unix]
-    - pango 1.40.*             # [linux]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - pkg-config               # [osx]
-    - zlib 1.2.11               # [unix]
+  host:
+    - cairo                    # [linux]
+    - expat                    # [linux]
+    - libpng                   # [unix]
+    - libtiff                  # [unix]
+    - libtool                  # [linux]
```

Comment:
Maybe we should start using this on macOS too? Looks like the `libtool` being used on macOS is running into [some issues]( https://travis-ci.org/conda-forge/graphviz-feedstock/builds/397342300 ).